.envrc

```
@@ -130,7 +130,7 @@ export LOGIN_GOV_ADMIN_CLIENT_ID="urn:gov:gsa:openidconnect.profiles:sp:sso:dod:
 export LOGIN_GOV_ENGADMIN_CLIENT_ID="urn:gov:gsa:openidconnect.profiles:sp:sso:dod:engadminmovemillocal"
 export LOGIN_GOV_HOSTNAME="idp.int.identitysandbox.gov"
 
-require LOGIN_GOV_SECRET_KEY "See 'chamber read app-devlocal login_gov_secret_key'"
+require LOGIN_GOV_SECRET_KEY "See 'AWS_REGION=us-gov-west-1 aws-vault exec transcom-gov-dev -- chamber read app-devlocal login_gov_secret_key'"
```

```suggestion
require LOGIN_GOV_SECRET_KEY "See 'DISABLE_AWS_VAULT_WRAPPER=1 AWS_REGION=us-gov-west-1 aws-vault exec transcom-gov-dev -- chamber read app-devlocal login_gov_secret_key'"
```
You'll need to add `DISABLE_AWS_VAULT_WRAPPER=1` to all of these before the region.

---

oof copy🍝  no good. fixing.